ext/citus.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, curl, lz4, zstd, krb5, icu, stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "12.0.0";
+
+  buildInputs = [ curl lz4 zstd krb5 icu.dev postgresql];
+
+  src = fetchFromGitHub {
+    owner  = "citusdata";
+    repo   = pname;
+    rev    = "refs/tags/v${version}";
+    hash   = "sha256-HH9/slsCRe2yIVIqwR8sDyqXFonf8BHhJhLzHNv1CF0=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{lib,share/postgresql/extension}
+
+    cp *.so      $out/lib
+    cp *.sql     $out/share/postgresql/extension
+    cp *.control $out/share/postgresql/extension
```

Comment:
Ok so it makes sense that this fails. Citus has actually two extensions (distributed and columnar inside it - two control files:

- https://github.com/citusdata/citus/blob/main/src/backend/distributed/citus--11.1-1.control
- https://github.com/citusdata/citus/blob/main/src/backend/columnar/citus_columnar.control

Will check how to fix this locally